### DISCUSSION TITLE
希望支持的功能

### DISCUSSION THREAD

**Discussion by Twshuai** [lollipopkit/flutter_server_box#33] [Ideas]
app内编辑文件，还有上传手机内的文件

**Comment by lollipopkit** [comment]
感谢参与讨论，上述功能都会有，但是编辑功能支持可能不完善。

近期在准备毕业，可能会比较慢。

:)

**Reply by Twshuai** [reply]
感谢佬回复，支持app👍👍
ios和安卓版本都在使用

**Comment by Twshuai** [comment]
> 感谢参与讨论，上述功能都会有，但是编辑功能支持可能不完善。
> 
> 近期在准备毕业，可能会比较慢。
> 
> :)

上传手机内文件好像不太行，安卓苹果都一样

**Reply by lollipopkit** [reply]
https://github.com/lollipopkit/flutter_server_box/assets/10864310/e21bf277-2c43-45fc-8df9-d7a68efe98dc

可能是权限不足等问题，换个路径试试

**Reply by Twshuai** [reply]
> https://github.com/lollipopkit/flutter_server_box/assets/10864310/e21bf277-2c43-45fc-8df9-d7a68efe98dc
> 
> 可能是权限不足等问题，换个路径试试

视频里的功能可以，从手机内向服务器上传文件不支持吧？

**Reply by lollipopkit** [reply]
指的是从手机文件管理器里选择文件上传？

**Reply by Twshuai** [reply]
> 指的是从手机文件管理器里选择文件上传？

是的是的
